### PR TITLE
Fix error handling and display in hex

### PR DESCRIPTION
**Pull request by AaronRobinsonMSFT**

### CODE REVIEW

File: src/Pktsniffer.java [hunk 1]
```diff
@@ -42,8 +42,18 @@ public static void main(String[] args) {
         pktsniffer.runSniffer();
     }
 
+    private static void usageAndExit(boolean isError) {
+        System.err.println("""
```
Comment (AaronRobinsonMSFT) [inline src/Pktsniffer.java:46]: Would you like this to be `System.out` instead?

Comment (Khronus14) [reply src/Pktsniffer.java:46]: No

File: src/Pktsniffer.java [hunk 2]
```diff
@@ -42,8 +42,18 @@ public static void main(String[] args) {
...
+                java Pktsniffer -r <file>
+                """);
+        System.exit(isError ? -1 : 0);
```
Comment (Khronus14) [inline src/Pktsniffer.java]:
```suggestion
        System.exit(isError ? 1 : 0);
```

**Review by Khronus14 — APPROVED** [review]

**Comment by AaronRobinsonMSFT** [comment]
W00t! Merge please.

**Pull request merged**